src/Post/Comments/NewComment.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import useForm from 'react-hook-form';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import './NewComment.css';
+
+const NewComment = ({ onSubmit }) => {
+
+    const [shouldDisplayNewComment, setShouldDisplayNewComment] = useState(false);
+    const {
+        register, handleSubmit,
+    } = useForm();
+
+    // declare event handler
+    const buttonClicked = () => setShouldDisplayNewComment(true);
+
+    if (shouldDisplayNewComment) {
+        return (
+            <form onSubmit={handleSubmit(onSubmit)} className="newComment">
+                <TextField
+                    id="standard-basic"
+                    label="New comment"
+                    multiline
+                    rowsMax="4"
+                    inputRef={register({ required: true })}
+                    name="newcomment"
```

Comment:
@opmanual **name** property set the key in the object that `react-forms-hook` returns when the form is submitted.

I think the name "comment" would be better here so that the object looks like

{
   commnet: "blah blah blah"
}